daemon/restapi/config.go

```diff
@@ -269,7 +269,7 @@ func (h *ConfigModifyEventHandler) changedOption(key string, value option.Option
 func (h *ConfigModifyEventHandler) triggerDatapathRegen(force bool, reason string) {
 	if force {
 		h.logger.Debug("PolicyEnforcement mode changed, increasing policy revision to enforce policy recalculation")
-		h.policy.BumpRevision()
+		h.policy.TriggerPolicyUpdates("PolicyEnforcement mode changed")
```

Comment:
This change causes a policy update event to be queued, whereas previously only the policy revision number was bumped. How does this queued work relate to the endpoint regen trigger below? Is there a possible race between the ensuing regeneration and the policy computation? I.e., what happens if the endpoint regeneration code gets to observe the old policy revision number, before the policy update event is served, and concludes that endpoint does not need to regenerate its policy?